hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithFSO.java

```diff
@@ -218,37 +223,47 @@ public class TestNSSummaryEndpointWithFSO {
           + CONTAINER_TWO_REPLICA_COUNT * BLOCK_TWO_LENGTH
           + CONTAINER_THREE_REPLICA_COUNT * BLOCK_THREE_LENGTH;
 
-  private static final long MULTI_BLOCK_KEY_SIZE_WITH_REPLICA
-          = LOCATION_INFO_GROUP_ONE_SIZE;
 
   private static final long LOCATION_INFO_GROUP_TWO_SIZE
       = CONTAINER_FOUR_REPLICA_COUNT * BLOCK_FOUR_LENGTH
       + CONTAINER_FIVE_REPLICA_COUNT * BLOCK_FIVE_LENGTH
       + CONTAINER_SIX_REPLICA_COUNT * BLOCK_SIX_LENGTH;
 
   private static final long FILE1_SIZE_WITH_REPLICA =
-      LOCATION_INFO_GROUP_ONE_SIZE;
+      getReplicatedSize(KEY_ONE_SIZE,
```

Comment:
Are we only considering file size with single replica?

Reply:
ohh yes! From the original testing setting it set up these files with the single replica!
(https://github.com/apache/ozone/blob/master/hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/OMMetadataManagerTestUtils.java#L218)

Reply:
Thanks for the explanation. Didn't catch it initially.